python/dataproc_templates/kafka/README.md

```diff
@@ -0,0 +1,63 @@
+# Kafka To BigQuery
+
+Template for reading files from Cloud Storage and writing them to a BigQuery table. It supports reading JSON, CSV, Parquet and Avro formats.
+
+It uses the 
+  - [Spark BigQuery connector](https://cloud.google.com/dataproc-serverless/docs/guides/bigquery-connector-spark-example) for writing to BigQuery.
+
+  - [Kafka-Clients Connector](https://mvnrepository.com/artifact/org.apache.kafka/kafka-clients) for connecting list of borker connections.
```

Comment:
I believe it doesn't require Kafka-Clients. The template requires [Kafka 0.10+ Source For Structured Streaming](https://mvnrepository.com/artifact/org.apache.spark/spark-sql-kafka-0-10), which transitively also requires Kafka-Clients.

Reply:
Changes made accordingly